aws/resource_aws_directory_service_directory_test.go

```diff
@@ -487,7 +558,7 @@ resource "aws_directory_service_directory" "connector" {
   type = "ADConnector"
 
   connect_settings {
-    customer_dns_ips = ["${aws_directory_service_directory.bar.dns_ip_addresses}"]
```

Comment:
I could not think of a quick way to support both Terraform 0.11 and Terraform 0.12 with this configuration (as its a set being returned), so opted to fix it for the future instead of this current error:

```
--- FAIL: TestAccAWSDirectoryServiceDirectory_connector (6.61s)
    testing.go:568: Step 0 error: errors during plan:

        Error: Incorrect attribute value type

          on /var/folders/v0/_d108fkx1pbbg4_sh864_7740000gn/T/tf-test119297193/main.tf line 24:
          (source code not available)

        Inappropriate value for attribute "customer_dns_ips": element 0: string
        required.
```